### PR TITLE
test: minor code formating

### PR DESCRIPTION
**Pull request by axkar**
Minor test formating including:
- new naming and indexing conventions
- ascii charts
- topology colors
- rename the "Initialize" step

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: test/case/ietf_interfaces/bridge_basic/topology.dot [hunk 1]
```diff
@@ -7,18 +7,18 @@ graph "1x2" {
...
         target [
-	    label="{ <mgmt> mgmt | <data> data } | target\n10.0.0.2/24 (br0)",
+	    label="{ <mgmt> mgmt | <data> data } | target",
 	    pos="10,12!",
 
 	    kind="infix",
 	];
 
-	host:tgt -- target:mgmt [kind=mgmt]
+	host:mgmt -- target:mgmt [kind=mgmt, color=grey]
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_basic/topology.dot:22]: color="lightgray"]

File: test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
-"""
+r"""
 Bridge forwarding dual DUTs
 
 Ping through two bridges on two different DUTs.
+
+....
+
+   ,-------------------------------------,       ,-------------------------------------,
+   |                          dut1:data2 |       | dut2:data2                          |
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py:10]: dut1:link dut2:link

File: test/case/ietf_interfaces/bridge_fwd_dual_dut/topology.dot [hunk 3]
```diff
@@ -7,32 +7,32 @@ graph "2x4" {
...
-	    height=1.2,
-	    kind="infix",
+		label="{ <mgmt> mgmt | <data1> data1 | <data2> data2 } | { <link> link | \n dut2 }",
+	    	pos="4,14.43!",
+	    	height=1.2,
+	    	kind="infix",
 	];
 
-	host:mgmt1 -- dut1:mgmt [kind=mgmt]
+	host:mgmt1 -- dut1:mgmt [kind=mgmt, color=grey]
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_fwd_dual_dut/topology.dot:30]: color="lightgray"]

File: test/case/ietf_interfaces/bridge_fwd_dual_dut/topology.dot [hunk 4]
```diff
@@ -7,32 +7,32 @@ graph "2x4" {
...
 	host:data11 -- dut1:data1 [color=black, fontcolor=black, taillabel="10.0.0.2"]
 
-	host:mgmt2 -- dut2:mgmt [kind=mgmt]
-	host:data20 -- dut2:data0 [color=black, fontcolor=black, taillabel="10.0.0.3"]
-	host:data21 -- dut2:data1 [color=black, fontcolor=black, taillabel="10.0.0.4"]
+	host:mgmt2 -- dut2:mgmt [kind=mgmt, color=grey]
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_fwd_dual_dut/topology.dot:33]: color="lightgray"]

File: test/case/ietf_interfaces/bridge_fwd_sgl_dut/test.py [hunk 5]
```diff
@@ -1,34 +1,40 @@
...
 
 Tests forwarding through a DUT with two bridged interfaces on one DUT.
+
+....
+
+,-----------------------------------------,
+|                                         |
+|                          br0            |
+|                         /   \           |
+| target:mgmt    tgt:data0     tgt:data1  |
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_fwd_sgl_dut/test.py:13]: target inte tgt

File: test/case/ietf_interfaces/bridge_fwd_sgl_dut/topology.dot [hunk 6]
```diff
@@ -7,19 +7,19 @@ graph "1x3" {
...
 
 	    kind="infix",
 	];
 
-	host:tgt -- target:mgmt [kind=mgmt]
-	host:data0 -- target:data0 [color=black, fontcolor=black, taillabel="10.0.0.1"]
-	host:data1 -- target:data1 [color=black, fontcolor=black, taillabel="10.0.0.2"]
-}
\ No newline at end of file
+	host:mgmt -- target:mgmt [kind=mgmt, color=grey]
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_fwd_sgl_dut/topology.dot:22]: color="lightgray"]

File: test/case/ietf_interfaces/igmp_vlan/test.py [hunk 7]
```diff
@@ -1,53 +1,54 @@
...
+
+....
+
+       VLAN55             VLAN77                          VLAN55              VLAN77
+      10.0.1.1           10.0.2.1                       10.0.1.2            10.0.2.2
+        \                    /                           \                     /
+         \                  /                             \                   /
+          \                /                               \                 /
+           \--------------/        VLAN 1,2 T               \---------------/
+           |    DUT1      +---------------------------------+      DUT2     |
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/igmp_vlan/test.py:15]: dut1:<what port>

File: test/case/ietf_interfaces/igmp_vlan/test.py [hunk 8]
```diff
@@ -1,53 +1,54 @@
...
+           |              |                                 |               |
+           +--------------+                                 +-----+---------+
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/igmp_vlan/test.py:17]: Same here, which port.

File: test/case/ietf_interfaces/igmp_vlan/topology.dot [hunk 9]
```diff
@@ -7,32 +7,32 @@ graph "2x4" {
...
+		label="{ <mgmt> mgmt | <data1> data1 | <data2> data2 } | { <link> link | dut2 \nVLAN55: 10.0.1.2 \nVLAN77: 10.0.2.2\n }",
+		pos="3,14.6!",
+		fontsize=12,
+		kind="infix",
 	];
 
-	host:mgmt1 -- dut1:mgmt [kind=mgmt]
-	host:data10 -- dut1:data0 [color=purple, fontcolor=purple, taillabel="10.0.1.11", headlabel="VLAN55 U"]
-	host:data11 -- dut1:data1 [color=darkgreen, fontcolor=darkgreen, taillabel="10.0.2.11", headlabel="VLAN77 U"]
+	host:mgmt1 -- dut1:mgmt [kind=mgmt, color=grey]
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/igmp_vlan/topology.dot:29]: color="lightgray"]

File: test/case/ietf_interfaces/igmp_vlan/topology.dot [hunk 10]
```diff
@@ -7,32 +7,32 @@ graph "2x4" {
...
+	host:data11 -- dut1:data1 [color=purple, fontcolor=purple, taillabel="10.0.1.11", headlabel="VLAN55 U"]
+	host:data12 -- dut1:data2 [color=darkgreen, fontcolor=darkgreen, taillabel="10.0.2.11", headlabel="VLAN77 U"]
 
-	host:mgmt2 -- dut2:mgmt [kind=mgmt]
-	host:data20 -- dut2:data0 [color=purple, fontcolor=purple, taillabel="10.0.1.22", headlabel="VLAN55 U"]
-	host:data21 -- dut2:data1 [color=darkgreen, fontcolor=darkgreen, taillabel="10.0.2.22", headlabel="VLAN77 U"]
+	host:mgmt2 -- dut2:mgmt [kind=mgmt, color=grey]
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/igmp_vlan/topology.dot:33]: color="lightgray"]

File: test/case/ietf_interfaces/verify_all_interface_types/topology.dot [hunk 11]
```diff
@@ -19,7 +19,7 @@ graph "1x3" {
 	    kind="infix",
 	];
 
-	host:tgt -- target:mgmt [kind=mgmt]
-	host:dummy0 -- target:ethX [color=black]
-	host:dummy1 -- target:ethQ [color=black]
+	host:mgmt -- target:mgmt [kind=mgmt, color=grey]
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/verify_all_interface_types/topology.dot:22]: color="lightgray"]

File: test/case/ietf_interfaces/verify_all_interface_types/topology.dot [hunk 12]
```diff
@@ -7,7 +7,7 @@ graph "1x3" {
 	edge [color="cornflowerblue", penwidth="2"];
 
 	host [
-	    label="host | { <tgt> tgt  | <dummy0> dummy0 | <dummy1> dummy1 }",
+	    label="host | { <mgmt> mgmt  | <dummy1> dummy1 | <dummy2> dummy2 }",
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/verify_all_interface_types/topology.dot:10]: data1 and data2

File: test/case/infix_services/services_basic/topology.dot [hunk 13]
```diff
@@ -19,7 +19,7 @@ graph "1x2" {
 	    kind="infix",
 	];
 
-	host:tgt -- target:mgmt [kind=mgmt]
+	host:mgmt -- target:mgmt [kind=mgmt, color=grey]
```
Comment (mattiaswal) [inline test/case/infix_services/services_basic/topology.dot:22]: color="lightgray"]

**Review by mattiaswal — CHANGES_REQUESTED** [review]
Great work, most of the comment is that use the same color as you used in OSPF test for MGMT 'lightgray' instead of 'gray'

**Review by troglobit — COMMENTED** [review]
Looks great, nice work! (I'll let the reviewers approve, just wanted to chime in :)

File: test/case/ietf_interfaces/bridge_basic/test.py [hunk 14]
```diff
@@ -1,18 +1,21 @@
...
 Bridge basic
 
 Test basic connectivity to a bridge
+
+....
+
+ PING -->         br0 (10.0.0.2)
+                 /
+   PC -------- e1
+....
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_basic/test.py:12]: target:data

File: test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py [hunk 15]
```diff
@@ -1,38 +1,45 @@
...
+   |                      br0  ----------|-------|---------  br0                       |
+   |                     /   \           |       |          /   \                      |
+   |dut1:mgmt  dut1:data0     dut1:data1 |       | dut2:data0    dut2:data1  dut2:mgmt |
+   '-------------------------------------'       '-------------------------------------'
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py:14]: Step up theese as well, so they map to the code

File: test/case/ietf_interfaces/bridge_fwd_sgl_dut/test.py [hunk 16]
```diff
@@ -1,34 +1,40 @@
...
+'-----------------------------------------'
+        |                |     |
+        |                |     |
+,------------------------------------------,
+|   host:mgmt   host:data0     host:data1  |
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_fwd_sgl_dut/test.py:18]: data0 == data1, data1 == data2

File: test/case/ietf_interfaces/igmp_vlan/test.py [hunk 17]
```diff
@@ -1,53 +1,54 @@
...
 Test tagged IGMP control traffic and that VLAN separation is respected for multicast
+
+....
+
+       VLAN55             VLAN77                          VLAN55              VLAN77
+      10.0.1.1           10.0.2.1                       10.0.1.2            10.0.2.2
+        \                    /                           \                     /
+         \                  /                             \                   /
+          \                /                               \                 /
+           \--------------/        VLAN 1,2 T               \---------------/
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/igmp_vlan/test.py:14]: Not your doing (its my from the beginning), but show the ports,  dut1:<port> dut2:<port> etc

File: test/case/ietf_interfaces/igmp_vlan/test.py [hunk 18]
```diff
@@ -1,53 +1,54 @@
...
+| msend +--+             | mreceive |                 | mreceive   |       | msend |
++-------+                +----------+                 +------------+       +--------+
+ 10.0.1.11                 10.0.2.11                      10.0.1.22         10.0.2.22
+
+....
+
 """
 
 import infamy
-import time
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/igmp_vlan/test.py:31]: Greaaaaat!! :1st_place_medal:

File: test/case/ietf_interfaces/bridge_basic/test.py
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_basic/test.py:1]: General comment: run make test-spec locally and check in the Readme.adoc for the test cases, this way you can look at the test case in github.

**Comment by mattiaswal** [comment]
> Looks great, nice work! (I'll let the reviewers approve, just wanted to chime in :)

You is also set as reviewer :+1:

File: test/case/ietf_interfaces/bridge_basic/topology.dot (hunk 1 above)
Comment (axkar) [reply test/case/ietf_interfaces/bridge_basic/topology.dot:22]: fixed

File: test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py (hunk 2 above)
Comment (axkar) [reply test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py:10]: fixed

File: test/case/ietf_interfaces/bridge_fwd_dual_dut/topology.dot (hunk 3 above)
Comment (axkar) [reply test/case/ietf_interfaces/bridge_fwd_dual_dut/topology.dot:30]: fixed

File: test/case/ietf_interfaces/bridge_fwd_sgl_dut/test.py (hunk 5 above)
Comment (axkar) [reply test/case/ietf_interfaces/bridge_fwd_sgl_dut/test.py:13]: fixed

File: test/case/ietf_interfaces/igmp_vlan/test.py (hunk 8 above)
Comment (axkar) [reply test/case/ietf_interfaces/igmp_vlan/test.py:17]: fixed

File: test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py (hunk 15 above)
Comment (axkar) [reply test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py:14]: fixed

File: test/case/ietf_interfaces/igmp_vlan/test.py (hunk 17 above)
Comment (axkar) [reply test/case/ietf_interfaces/igmp_vlan/test.py:14]: fixed

File: test/case/ietf_interfaces/bridge_basic/test.py
Comment (axkar) [reply test/case/ietf_interfaces/bridge_basic/test.py:1]: Thnx, I'll do that

File: test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py [hunk 19]
```diff
@@ -1,38 +1,45 @@
...
+   ,-------------------------------------,       ,-------------------------------------,
+   |                          dut1:link  |       | dut2:link                           |
+   |                      br0  ----------|-------|---------  br0                       |
+   |                     /   \           |       |          /   \                      |
+   |dut1:mgmt  dut1:data1     dut1:data2 |       | dut2:data1    dut2:data2  dut2:mgmt |
+   '-------------------------------------'       '-------------------------------------'
+       |                |     |                            |     |                 |
+       |                |     |                            |     |                 |
+,-----------------------------------------------------------------------------------------,
+|  host:mgmt0  host:data10    host:data11         host:data20    host:data21   host:mgmt1 |
```
Comment (mattiaswal) [inline test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py:18]: host:data20 does not exist

File: test/case/ietf_interfaces/vlan_ping/topology.png
Comment (mattiaswal) [inline test/case/ietf_interfaces/vlan_ping/topology.png:1]: Unclear why your computer makes diff on the png files, but do not add the one you not changed the test for.

**Review by mattiaswal — CHANGES_REQUESTED** [review]

File: test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py (hunk 19 above)
Comment (axkar) [reply test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py:18]: yes, and similar to the left...

Comment (axkar) [reply test/case/ietf_interfaces/bridge_fwd_dual_dut/test.py:18]: modified mgmt, too

File: test/case/ietf_interfaces/vlan_ping/topology.png
Comment (axkar) [reply test/case/ietf_interfaces/vlan_ping/topology.png:1]: I agree, very strange. But I reverted all the .png which I didn't touch

**Review by troglobit — APPROVED** [review]

**Review by mattiaswal — APPROVED** [review]
Great :medal_sports:

**Pull request merged**